pyhms/demes/cma_deme.py

```diff
@@ -27,8 +82,16 @@ def __init__(
         if random_seed is not None:
             opts["randn"] = np.random.randn
             opts["seed"] = random_seed + self._started_at
+        if config.sigma0:
+            self._cma_es = CMAEvolutionStrategy(x0.genome, config.sigma0, inopts=opts)
+        elif config.__dict__.get("set_stds"):
```

Comment:
Probably I should add new field to the CMADemeConfig

Reply:
As the warm-start is not a straight up improvement in every situation maybe leave it as a keyword argument option intended more for our internal use.